### ISSUE TITLE
Static covariates in probabilistic regression models

### ISSUE THREAD

**Issue by tadky** [unit8co/darts#1604]
Hi, I am new to Darts and I wanted to use static covariates in probabilistic regression models, but as the dimensions of static cov and number of samples for given points are different it fails to concatenate. Am I missing something or is it just not supported. There is max_samples_per_ts argument which seems to me that it could solve it, but it is hard set to 1 and probably there is some problem afterwards otherwise it would be implemented.  Thank you.

PS. I didn't know where to ask/write, so I hope it's ok

**Comment by dennisbader** [comment]
Hi @tadky, without any code to reproduce it is hard to answer this question.

- [here](https://unit8co.github.io/darts/examples/15-static-covariates.html) is an example on how to use static covariates (the example uses TFTModel for forecasting, but you can swap it with one of our Regression Models)
- `max_samples_per_ts` will not help here. This just sets how many samples to extract from your input data to build the training set.

**Comment by tadky** [comment]
Thank you for replying, unfortunately it is hard to send code as it is not some simple script. But I will try to make small example.


```
ts = TimeSeries.from_dataframe(df, "Date", "High")
ts = ts.with_static_covariates(pd.DataFrame({'test': [0]}))
train, val = ts.split_before(0.7)
model = LinearRegressionModel(lags=7, likelihood = "quantile", quantiles=[0.01, 0.1, 0.5, 0.9, 0.99])
model.fit(train)
ts_val = model.predict(len(val), num_samples=100, series=train)
```

And that produces this following exception.

```  
File "/home/tadky/.virtualenvs/DF/lib/python3.8/site-packages/darts/utils/data/tabularization.py", line 308, in_add_static_covariates
   return np.concatenate([features, static_covs], axis=1)
File "<__array_function__ internals>", line 5, in concatenate
ValueError: all the input array dimensions for the concatenation axis must match exactly, but along dimension 0, the array at index 0 has size 100 and the array at index 1 has size 1```

**Comment by dennisbader** [comment]
Hey @tadky, this was fixed in #1469 and will come with the next darts release in the coming weeks!

**Comment by dennisbader** [comment]
@FrederikSchibelfeldt , I'm not sure I understand your issue. It seems to me like this is unrelated to the current issue. 

Could you open a dedicated issue with a minimum reproducible example (also is this darts related or rather pandas/numpy)?
